src/test/resources/application-test.yml

```diff
@@ -60,6 +60,8 @@ courtsentencing:
   page:
     size: 10
   has-migrated-data: true
+finance:
+  transactions.forwardingDelaySeconds: 1
```

Comment:
you might want to consider moving this to 0 - else I guess every test will take at least 1 second

Reply:
One problem is that the multi-event test can fail so Ive separated that into another test class so i can set its forwardingDelaySeconds back to 1.